code/modules/hydroponics/grown/flowers.dm

```diff
@@ -15,7 +15,7 @@
 	icon_grow = "poppy-grow"
 	icon_dead = "poppy-dead"
 	mutatelist = list(/obj/item/seeds/poppy/geranium, /obj/item/seeds/poppy/lily)
-	reagents_add = list("bicaridine" = 0.2, "nutriment" = 0.05)
+	reagents_add = list("bicaridine" = 0.2, "plantmatter" = 0.05)
```

Comment:
Could replace them with something safer like sal glu or another safer existing chem, point was it seemed strange to re-add the chems